src/Covenant/Internal/Rename.hs

```diff
@@ -188,7 +293,7 @@ renameCompT (CompT abses (CompTBody xs)) = RenameM $ do
 --
 -- @since 1.0.0
 renameValT :: ValT AbstractTy -> RenameM (ValT Renamed)
-renameValT = \case
+renameValT v = case v of
```

Comment:
Any reason why you need a named variable here?

Reply:
Artifact of the traces.